src/lib/kw_db.sh

```diff
@@ -232,6 +233,59 @@ function select_from()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function gets the values in the table of given database
+# with the given conditions
+#
+# @table:     Table to select info from
+# @columns:   Columns of the table to get
+# @pre_cmd:   Pre command to execute
+# @_condition_array: An array reference of condition pairs
+# @db:        Name of the database file
+# @db_folder: Path to the folder that contains @db
+#
+# Return:
+# 2 if db doesn't exist; 22 if table is empty
+# 0 if succesful; non-zero otherwise
+function select_from_where()
```

Comment:
This commit should be one of the first commits in your series.
@davidbtadokoro could you also check this commit?